doc/modules/linear_model.rst

```diff
@@ -422,24 +421,24 @@ Least Angle Regression
 Least-angle regression (LARS) is a regression algorithm for
 high-dimensional data, developed by Bradley Efron, Trevor Hastie, Iain
 Johnstone and Robert Tibshirani. LARS is similar to forward stepwise
-regression. At each step, it finds the predictor most correlated with the
-response. When there are multiple predictors having equal correlation, instead
-of continuing along the same predictor, it proceeds in a direction equiangular
-between the predictors.
+regression. At each step, it finds the feature most correlated with the
+target. When there are multiple features having equal correlation, instead
+of continuing along the same feature, it proceeds in a direction equiangular
+between the features.
 
 The advantages of LARS are:
 
   - It is numerically efficient in contexts where p >> n (i.e., when the
-    number of dimensions is significantly greater than the number of
-    points)
+    number of features is significantly greater than the number of
```

Comment:
use n_features and n_samples and remove the parenthesis maybe

Reply:
Removed the variables altogether, writing it in words instead.